Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -121,122 +179,6 @@ uint64_t AODProducerWorkflowDPL::getTFNumber(uint64_t firstVtxGlBC, int runNumbe
   return ts;
```

Comment:
No sure this is correct. 1st of all, what you extract is not the TFID but the timestamp.
But even assuming that this is what you want to get: the ts initially is the timestamp of SOR in \mus. Then you add to it the time difference between the 1st vertex and 1st seen BC, both withing the same TF (which can be anywhere within the run). This time difference might be 0 and limited in time by the duration of the TF. So, you will always get the timestamp of "SOR + something<TF duration".
I think you should have something like:
````
// to get the timestamp
ts += (firstRec - initialOrbit).bc2ns() / 1000000; 
````
or
````
// to get the TFid since the beginning of run
auto tfID = firstRec.differenceInBC(initialOrbit) / o2::constants::lhc::LHCMaxBunches / nHBFperTF;
````